### PR TITLE
[CPM- 94] seed dummy data for students

### PR DESCRIPTION
**Pull request by HongShi10**

### CODE REVIEW

File: seed-db/scripts/students.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+const createStudent = () => {
+    const firstName = faker.name.firstName();
+    const lastName = faker.name.lastName();
+    const upi = firstName.substring(0, 1).toLowerCase() + lastName.substring(0, 3).toLowerCase() + getRandomInt(100, 999);
```
Comment (daltonlim) [inline seed-db/scripts/students.js:21]: bro abstract into function

**Review by daltonlim — CHANGES_REQUESTED** [review]

File: seed-db/scripts/students.js (hunk 1 above)
Comment (HongShi10) [reply seed-db/scripts/students.js:21]: what

Comment (daltonlim) [reply seed-db/scripts/students.js:21]: nvm thought it was something else

File: docker-compose.yml [hunk 2]
```diff
@@ -1,40 +1,40 @@
-version: "3"
```
Comment (daltonlim) [inline docker-compose.yml:1]: Can you reclone repo and reapply your changes. The files are being uploaded as CLRF.

**Review by daltonlim — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (HongShi10) [reply docker-compose.yml]: should be fixed

**Comment by daltonlim** [comment]
LGTM

**Review by daltonlim — APPROVED** [review]
Lgtm

**Review by FocalChord — APPROVED** [review]
LGTM

**Pull request merged**